build/scripts-3.4/pydeface.py

```diff
@@ -0,0 +1,180 @@
+#!/opt/local/bin/python
```

Comment:
This is legacy from a MacPorts python installation. Not sure whether it's mine or Russ'. Either way, should be safe to update to a more linuxy location (eg #!/usr/bin/env python3)